maths/decimal_isolate.py

```diff
@@ -0,0 +1,25 @@
+"""
+Isolate the Decimal part of a Number
+"""
```

Comment:
Please provide a URL to a document describing this algorithm and why it is useful.